src/styles/blocks/calendar.scss

```
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: #{ $day__size * 7 + $day-gap * 6};
```

Здесь не хватает 4px, выясни почему?
```suggestion
  width: #{ $day__size * 7 + $day-gap * 6};
```